packages/bsky/src/feature-gates.ts

```diff
@@ -0,0 +1,67 @@
+import { Statsig, StatsigUser } from 'statsig-node'
+import { sha256Hex } from '@atproto/crypto'
+
+export type Config = {
+  apiKey?: string
+  env?: 'development' | 'staging' | 'production' | string
+}
+
+export enum GateID {
+  NewSuggestedFollowsByActor = 'new_sugg_foll_by_actor',
+}
+
+/**
+ * @see https://docs.statsig.com/server/nodejsServerSDK
+ */
+export class FeatureGates {
+  ready = false
+  private statsig = Statsig
+  ids = GateID
+
+  constructor(private config: Config) {}
+
+  async start() {
+    try {
+      if (this.config.apiKey) {
+        await this.statsig.initialize(this.config.apiKey, {
+          environment: {
+            tier: this.config.env || 'development',
+          },
+        })
+        this.ready = true
+      } else if (process.env.NODE_ENV === 'test') {
+        /**
+         * @see https://docs.statsig.com/server/nodejsServerSDK#local-overrides
+         */
+        await this.statsig.initialize('secret-key', {
+          localMode: true,
+          environment: {
+            tier: 'test',
+          },
+        })
+        this.ready = true
+      }
+    } catch (_) {
+      this.ready = false
```

Comment:
We should at minimum log the error so we can see there's an issue—you'll find some loggers in `./logger` and we can add a new one if we want!